server/models/PayoutMethod.ts

```diff
@@ -213,6 +213,17 @@ class PayoutMethod extends Model<InferAttributes<PayoutMethod>, InferCreationAtt
       return [];
     }
   }
+
+  async canBeEditedOrDeleted(): Promise<boolean> {
+    const expenses = await (sequelize.models.Expense as typeof Expense).findOne({
+      where: {
+        PayoutMethodId: this.id,
+        status: { [Op.notIn]: [ExpenseStatuses.PENDING, ExpenseStatuses.DRAFT] },
```

Comment:
Do you think it would make sense to include REJECT or CANCELED as well?

Reply:
I think the question here was related to what status I know for sure we don't care about editing. With the existing expense flow, I wonder if something would end up in canceled before being edited to the correct state. But perhaps if the user submits to the wrong collective and needs to re-submit to another collective and change something in the Payout Method details, that would make sense.